src/main/java/networkbook/logic/commands/UndoCommand.java

```diff
@@ -0,0 +1,21 @@
+package networkbook.logic.commands;
+
+import networkbook.logic.commands.exceptions.CommandException;
+import networkbook.model.Model;
+
+/**
+ * Clears the network book.
+ */
+public class UndoCommand extends Command {
+
+    public static final String COMMAND_WORD = "undo";
+    public static final String MESSAGE_SUCCESS = "Noted, previous state change undone!";
+    public static final String MESSAGE_UNDO_DISALLOWED = "Illegal state change requested.\n"
+            + "NetworkBook does not have a previous state stored to undo to.";
```

Comment:
I'm not sure if this sentence is to technical, can be better phrased to "You have not made any change to undo?"